### DISCUSSION TITLE
Multi-Client handling

### DISCUSSION THREAD

**Discussion by WolfgangFahl** [elimintz/justpy#557] [Ideas]
Justpy can handle the situation below easily. On data update different views are changed:
![grafik](https://user-images.githubusercontent.com/1336221/191026115-b9c33078-178b-40a9-8750-d8ab1c7c34af.png)

But what if the three views are by different users on different clients/browsers.
It would still be possible but more complicated since the common state of all clients would have be shared e.g. by a central blackboard/repository such as a database or file. I think it would be great to create more example use cases for a multi client scenario since this is where other approaches are week or complicated. A strong simple solution would IMHO be a USP for justpy.

**Comment by elimintz** [comment]
Are you looking for an example in which the same data is shown using different charts to different users? Or the case where each user has their own data?
